### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by rmulhol**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -3,63 +3,31 @@ module github.com/vulcanize/vulcanizedb
...
-	github.com/ethereum/go-ethereum v1.9.0
-	github.com/fjl/memsize v0.0.0-20190710130421-bcb5799ab5e5 // indirect
-	github.com/gballet/go-libpcsclite v0.0.0-20190607065134-2772fd86a8ff // indirect
-	github.com/go-sql-driver/mysql v1.4.1 // indirect
-	github.com/golang/protobuf v1.3.2 // indirect
-	github.com/google/uuid v1.0.0 // indirect
-	github.com/graph-gophers/graphql-go v0.0.0-20190610161739-8f92f34fc598 // indirect
-	github.com/hashicorp/golang-lru v0.0.0-20180201235237-0fb14efe8c47
-	github.com/howeyc/fsnotify v0.9.0 // indirect
+	github.com/ethereum/go-ethereum v1.9.1
```
Comment (elizabethengelman) [inline go.mod]: 💯

**Comment by rmulhol** [comment]
> This LGTM. Just curious what the impetus for updating some of these deps is, besides make sure we're have the most recent versions.
> 
> I wonder if a similar update needs to made to transformer repos too. 🤔Locally I think I've seen `plugin was built with a different version of package...` errors if vdb deps don't match up with transformer deps (specifically the geth version).

Yeah the main impetus is to bring it in line with current transformer repos. Basically trying to sidestep exactly that error while working on a more streamlined deploy

**Comment by elizabethengelman** [comment]
@rmulhol gotcha! thanks for explanation - I'm running into this sort of issue with `storage-diffs-over-rpc` right now as well.

I wonder if there's an easier way to keep the deps in these two repos in line 🤔

**Comment by rmulhol** [comment]
@elizabethengelman I hope so! Thinking it should be possible to say that a versioned release on this repo should be the source of truth for versions of shared dependencies, but not yet sure how to enforce that 🤔

**Pull request merged**